### PR TITLE
Set up Testing Ci for main branch

### PR DESCRIPTION
**Pull request by DevangRaval1**
@nvkulkarni12

### CODE REVIEW

File: frontend/src/__tests__/donor.test.tsx
Comment (sourabhk25) [inline frontend/src/__tests__/donor.test.tsx:1]: This test case is repeated. Please remove the redundant code.

**Review by sourabhk25 — CHANGES_REQUESTED** [review]
Please remove repeated testcase in donor.test.tsx

File: frontend/src/__tests__/donor.test.tsx
Comment (DevangRaval1) [reply frontend/src/__tests__/donor.test.tsx:1]: Done

**Review by sachinprasad1998mav — APPROVED** [review]
LGTM

**Review by sourabhk25 — APPROVED** [review]
LGTM - 

image

image


image

**Review by rohitkarhadkar — APPROVED** [review]
LGTM!!

**Review by mmendan — APPROVED** [review]
Looks good to me!

**Review by prasannarajezzzy — APPROVED** [review]

**Pull request merged**